src/main/java/com/cloudbees/jenkins/support/impl/SlaveCommandStatistics.java

```diff
@@ -56,7 +65,14 @@
 @Extension
 public final class SlaveCommandStatistics extends Component {
 
+    /*protected*/ static @Nonnegative int MAX_STATS_SIZE = 1000;
```

Comment:
Would it be interesting to allow customizing this value through a system property?

Reply:
I guess it can always be added if someone feels the need. For now, clearly, noone had a reason to even consider it since we seem to be the first ones running into this problem.